clusterloader2/pkg/measurement/util/wait_for_pods.go

```diff
@@ -39,6 +40,10 @@ type WaitForPodOptions struct {
 	EnableLogging       bool
 	CallerName          string
 	WaitForPodsInterval time.Duration
+
+	// UpdatedPodPredicate can be used to detect which pods have been already updated.
+	// nil value means all pods are updated.
+	UpdatedPodPredicate func(_ *v1.Pod) bool
```

Comment:
NIT: For me more readable is "IsPodUpdated[Predicate]" but it might be matter of taste

Reply:
Thanks